### PR TITLE
[bitnami/postgresql-ha] Fix maxPool in values.yaml

### PR DESCRIPTION
**Pull request by dominik-bln**
**Description of the change**

`values.yaml` contains a wrong key for `maxPool`

**Additional information**

**Checklist** 
- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [x] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

:warning: Keep in mind that if you want to make changes to the kubeapps chart, please implement them in the [kubeapps repository](https://github.com/kubeapps/kubeapps/tree/master/chart/kubeapps). This is only a synchronized mirror.

### CODE REVIEW

**Review by miguelaeh — APPROVED** [review]
Thank you very much for the PR!!!
LGTM!!!

**Comment by miguelaeh** [comment]
Since it is a WIP PR I will wait until you change the state to merge it.
Regards

**Comment by dominik-bln** [comment]
@miguelaeh It's marked as ready now, I just forgot to remove the flag.

**Pull request merged**
